queues/one_to_one_test.go

```diff
@@ -124,15 +121,17 @@ func Benchmark1Producer1ConsumerOneRing(b *testing.B) {
 	startNanos := make([]int64, b.N)
 	endNanos := make([]int64, b.N)
 
-	var ring onering.SPSC
-	ring.Init(8192)
+	var ring = onering.New{Size:8192}.SPSC()
 	var wg sync.WaitGroup
 	wg.Add(2)
-
+	var nums = make([]int64, b.N)
```

Comment:
Why a premade integer array in these benchmarks but not in any others? This seems like a difference between benchmarks?

Reply:
Since onering can only send pointers now without allocating memory, this seemed like a fair change (that or removing a check for value in the receiver). I'll leave it up to you to decide if it was of course. The assumption is that no one should pass non pointers to onering unless being explicitly ok with there being a malloc on every send. I put a disclaimer to onering page about it + added an instrumentation to detect it and warn about it: https://github.com/pltr/onering/blob/master/debug.go#L61

Reply:
Do you think we should alter the others to do the same thing? I want to make sure we are comparing apples to apples. If they can do the same, we should make them all the same I think.

Thoughts?

Reply:
Good point